mk/spksrc.service.mk

```diff
@@ -39,7 +39,7 @@
 #                                           folder at intallation and runtime.
 #  SSS_SCRIPT                    (optional) custom script file for service start/stop/status when the generic 
 #                                           installer generated script (SERVICE_SETUP) is not usable.
-#  NO_SERVICE_SHORTCUT           (optional) do not create 
+#  NO_SERVICE_SHORTCUT           (optional) do not create an app icon in the DSM desktop
```

Comment:
this is just an update of the comment and has to be validated for correct implementation.